packages/p2p-v2/src/pages/my-profile/screens/MyProfileCounterparties/MyProfileCounterpartiesTable.scss

```diff
@@ -0,0 +1,44 @@
+.p2p-v2-my-profile-counterparties-table {
+    overflow-y: auto;
+    height: 28rem;
+    display: flex;
+    gap: 0.8rem;
+    flex-direction: column;
+    width: 100%;
+    position: absolute;
+
+    @include mobile {
+        height: 100%;
+    }
+
+    &::-webkit-scrollbar {
+        width: 5px;
+        height: 5px;
+        background-color: transparent;
+        border-radius: 10px;
+    }
+
+    &::-webkit-scrollbar-track {
+        background-color: transparent;
+    }
+
+    &::-webkit-scrollbar-thumb {
+        display: none;
+    }
+
+    &:hover {
+        &::-webkit-scrollbar-thumb {
+            display: unset;
+            border-radius: 10px;
+            background-color: var(--state-active);
+        }
```

Comment:
for the scrollbar styling, if we move it to the root scss file, will it be applied to all scrollbars?

Reply:
for some reason, its getting overridden by the root styles and also in our current production i observed that the scrollbar styles have changed.. no idea if its related to the chrome update or any changes in our code.. need to check on it.

Reply:
ah i see, no worries then can resolve this comment